Kubeadm_Installation_Using_Containerd.md

```
@@ -0,0 +1,146 @@
+
+**On Master Node:**
+
+```bash
+kubectl get nodes
+
+
+
+
+```
+
+   <img src="https://raw.githubusercontent.com/faizan35/kubernetes_cluster_with_kubeadm/main/Img/nodes-connected.png" width="70%">
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)